src/graphql/Context/UserRequestContext.php

```diff
@@ -4,21 +4,21 @@
 
 class UserRequestContext
 {
-  private string|null $userId;
-  private string|null $tenantId;
+  private string $userId;
+  private string $tenantId;
```

Comment:
Application is using the tenantId at multiple places, where it can't be null.
Forcing this to be set, to avoid issues and strenghten security. We don't want a user to access data of another tenant.